### PR TITLE
Add user ID setter & getter methods to token object 

### PR DESCRIPTION
**Pull request by unfulvio**
# Summary

This PR adds a `get_user_id()` and a `set_user_id()` public methods to the `SV_WC_Payment_Gateway_Payment_Token` object.

### QA

- [ ] When using `set_user_id( (int) $user_id )` a `user_id` should be added to the `SV_WC_Payment_Gateway_Payment_Token::$data` array.
- [ ] When using `get_user_id()` a `user_id` should be returned as it appears in the `SV_WC_Payment_Gateway_Payment_Token::$data` array.

### CODE REVIEW

File: woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php [hunk 1]
```diff
@@ -136,6 +137,32 @@ public function get_id() {
...
 
 
+	/**
+	 * Gets the ID of the user associated with the token.
+	 *
+	 * @since 5.6.0-dev.1
+	 *
+	 * @return int
+	 */
+	public function get_user_id() {
```
Comment (asseti6) [inline woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php]: @unfulvio these should be two separate commits to be atomic the way we have been briefly discussing

**Comment by unfulvio** [comment]
hey @deivamagalhaes sorry that was a typo on my end (probably I copypasted in the QA instructions the equivalent for gateway_id which does take a string) - user IDs in WooCommerce/WordPress are integers so it's fine here - thanks
